fuzzers/aflsmart/runner.Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/fuzzbench/base-runner
+
+RUN apt-get install mono-complete -y && \
```

Comment:
I think (but I'm not sure) that not doing `apt-get update` in this file can cause issues if the parent image is cached.
Also, nit: please install packages in one invocation of `apt-get`.

Reply:
Yes, I added apt-get update. Thanks